README.md

```diff
@@ -44,4 +57,4 @@ your local Drupal installation for SIMPLETEST_BASE_URL.
     cd /path/to/drupal-8/core
     export SIMPLETEST_DB=mysql://drupal-8:password@localhost/drupal-8
     export SIMPLETEST_BASE_URL=http://drupal-8.localhost
-    ../vendor/bin/phpunit ../modules/typed_data
+    ../vendor/bin/phpunit ../modules/typed_data/tests/src/Unit/ActionSetTest.php
```

Comment:
I doubt ActionSetTest will work for typed data - this can stay with the old command.